Formula/cockroach-sql.rb

```diff
@@ -5,6 +5,10 @@ class CockroachSql < Formula
   desc "Distributed SQL database shell"
   homepage "https://www.cockroachlabs.com"
   version "23.1.11"
+  on_linux do
+    url "https://binaries.cockroachdb.com/cockroach-sql-v23.1.11.linux-amd64.tgz"
+    sha256 "8a42076aa8d4448820eb47d9816d08f110d830b263e7edfab1e57f354e6817bb"
```

Comment:
This file is generated, see the top comment.
In order to add Linux, you need to edit https://github.com/cockroachdb/homebrew-tap/blob/master/release/main.go to include the new platform.
Also, you proably want to separate amd64 and arm64 architectures. And the file needs a bit of refactoring to become more readable with so many platforms.